app/index.jsx

```diff
@@ -54,10 +54,11 @@ function getServerUrl() {
 const chartId = '1';
 const appId  = CIQ.localStorage.getItem('config.app_id') || 12812;
 const serverUrl = getServerUrl();
+const language = new URLSearchParams(window.location.search).get('l') || getLanguageStorage();
```

Comment:
@brucebinary I think `URLSearchParams` may not work on `Edge 15` and `Safari 9`, would you please check? if yes, what is your idea? change our support version or make custom function beside this one?

Reply:
We're not supporting those right? [This is our browserlist setting](https://browserl.ist/?q=defaults%2C+not+op_mini+all%2C+not+ie+%3C%3D+11%2C+not+IE_Mob+%3C%3D+11%2C+not+Samsung+%3C%3D+4)

Reply:
ok just talked with binary-static team. We need to support all edge versions.